### PR TITLE
Use HTTPS URL when possible

### PR DESCRIPTION
**Pull request by m1guelpf**
GitHub Issue (If applicable): N/A

## PR Type

What kind of change does this PR introduce?
- Documentation content changes

## What is the current behavior?

Some README links use an HTTP version of the link (and some websites don't redirect to the HTTPS version automatically), resulting in possible security issues (like a MITM attach) for the users who visit them.

## What is the new behavior?
Switched up all the links that had an HTTPS version

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Tested code with current [supported SDKs](https://github.com/unoplatform/uno/blob/master/README.md#uno-features)
- [x] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [x] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [x] [Wasm UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md#running-the-webassembly-ui-tests-snapshots) are not showing unexpected any differences. Validate PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [x] Updated the [Release Notes](https://github.com/unoplatform/uno/tree/master/doc/ReleaseNotes)
- [ ] Associated with an issue (GitHub or internal)

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -108,7 +108,7 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
...
+    <td align="center"><a href="https://www.dissolutegames.com/"><img src="https://avatars0.githubusercontent.com/u/8270914?v=4" width="100px;" alt="David Oliver"/><br /><sub><b>David Oliver</b></sub></a><br /><a href="#blog-davidjohnoliver" title="Blogposts">📝</a> <a href="https://github.com/unoplatform/uno/commits?author=davidjohnoliver" title="Code">💻</a> <a href="#content-davidjohnoliver" title="Content">🖋</a> <a href="https://github.com/unoplatform/uno/commits?author=davidjohnoliver" title="Documentation">📖</a> <a href="#maintenance-davidjohnoliver" title="Maintenance">🚧</a> <a href="#tutorial-davidjohnoliver" title="Tutorials">✅</a> <a href="#review-davidjohnoliver" title="Reviewed Pull Requests">👀</a> <a href="https://github.com/unoplatform/uno/commits?author=davidjohnoliver" title="Tests">⚠️</a> <a href="#example-davidjohnoliver" title="Examples">💡</a></td>
```
Comment (jeromelaban) [inline README.md]: Do not modify this section, it is generated content, it will be overwritten.

**Comment by jeromelaban** [comment]
Thank you for your contribution !

File: README.md (hunk 1 above)
Comment (ghuntley) [reply README.md]: Agreed. The rest of the PR is good to go, so I'll merge this as-is and then next time we add a contributor to the documentation it'll undo this.

**Comment by ghuntley** [comment]
@all-contributors please add @m1guelpf for docs

**Comment by ghuntley** [comment]
And ^^ will undo the changes in the contributors section ;-)

**Pull request merged**